srcpkgs/font-spleen/template

```diff
@@ -42,5 +42,8 @@ do_install() {
 	done
 	vinstall fonts.alias 644 usr/share/fonts/X11/spleen
 
+	for f in *.otb; do
```

Comment:
could the loop in line 40 be turned into `for f in *.pcf.gz *.otb; do`?

Reply:
Sure! I wasn't sure if it made sense to put the otb versions in the same directory, but I haven't had any problems doing that so your suggestion sounds good to me.